src/schema.py

```diff
@@ -87,7 +88,26 @@
 CREATE UNIQUE INDEX bayesdb_generator_i_default ON bayesdb_generator (tabname)
     WHERE defaultp;
 '''
-
+
+bayesdb_schema_upgrade[6] = '''
+PRAGMA user_version = 7;
+
+CREATE TABLE bayesdb_session (
+	id		INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT
+				CHECK (0 < id),
+	sent		BOOLEAN DEFAULT 0
+);
+CREATE TABLE bayesdb_session_entries (
+	id		INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT
+				CHECK (0 < id),
+	session_id	INTEGER NOT NULL REFERENCES bayesdb_session(id),
+	time		INTEGER NOT NULL, -- Unix time.
+	type		TEXT,
```

Comment:
Date: Tue, 06 Oct 2015 15:56:30 -0700
   From: Marco Cusumano-Towner <notifications@github.com>

   > +CREATE TABLE bayesdb_session_entries (
   > +	id		INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT
   > +				CHECK (0 < id),
   > +	session_id	INTEGER NOT NULL REFERENCES bayesdb_session(id),
   > +	time		INTEGER NOT NULL, -- Unix time.
   > +	type		TEXT,

   Done. Because sqlite does not have an ENUM type, I used this instead:
           type            TEXT CHECK (type IN ('bql','sql')) NOT NULL,

The standard way to do this is to introduce a new table containing all
the session types with numeric identifiers, and use a foreign key
reference.  See, e.g., the bayesdb_column_stattype table.